aspnetcore/blazor/globalization-localization.md

```diff
@@ -149,6 +149,19 @@ Localization is handled by the app in the following sequence of events:
 1. The Localization Middleware reads the cookie and assigns the culture.
 1. The Blazor Server session begins with the correct culture.
 
+When working with a <xref:Microsoft.AspNetCore.Mvc.Razor.RazorPage>, use the <xref:Microsoft.AspNetCore.Mvc.Razor.RazorPage.Context> property:
+
+```razor
+@{
+    this.Context.Response.Cookies.Append(
```

Comment:
Out of curiosity, do our docs tell users to avoid putting non-UI specific code in the Page? It's somewhat preferrable to put this in the `OnGet` method.

Reply:
These docs don't for this purpose ... just in `_Host.cshtml` above at Line 123, not sure about in the Razor syntax or Razor Pages overview topics.

Do you want to proceed with this? I'm not really sure why we're doing this in the first place. Isn't `_Host.cshtml` setting `PageBase.HttpContext`? Isn't that all that's needed here?

Reply:
> I'm not really sure why we're doing this in the first place.

Sorry, I'm lost now. Why do you think so?

Reply:
What's the use case for *not* using the `Pages/_Host.cshtml` file for this (given that this is for Blazor docs) as described in the preceding content (Line 123 down).

Anyway ... if we're keeping this additional content, do you want me to refactor it for `OnGet` in a `PageModel`? ... or show both with perhaps "preferred" language added to the description for the `OnGet` approach?

Reply:
@pranavkm ... Should I swap for `OnGet`/`PageModel` or **_add_** that scenario here? If not, then this PR is ready to go in.

Reply:
Yeah, let's keep this. It's a personal preference \ code hygiene where we avoid telling users to tweak the Request \ Response directly as part of a Razor Page or a View.